ProjetSupaero2018/catkin_ws/src/display/models/basic_car/model.sdf

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" ?>
+<sdf version="1.5">
+	<model name="basic_car">
+	  <link name="sphere">
+		<pose>0 0 1 0 0 0</pose>
+		
+		<inertial>
```

Comment:
Do we really need an inertia model ?

Reply:
Indeed we don't, but I believe it might be compulsory